api/main.py

```diff
@@ -10,7 +10,10 @@
 
 from typing import List
 from fastapi import Depends, FastAPI, HTTPException, status, Request
-from fastapi.security import OAuth2PasswordRequestForm
+from fastapi.security import (
+    OAuth2PasswordRequestForm,
+    SecurityScopes
+)
```

Comment:
`SecurityScopes` is not used in `api.main` so the original imports can be left unchanged.

Reply:
It has been used by `main.get_current_user`.

```
async def get_current_user(
        security_scopes: SecurityScopes,
        token: str = Depends(auth.oauth2_scheme)):
```

Reply:
Ah OK, I think that change is in a separate commit than the one where the import is being added.

Reply:
I have updated the commits. Thanks.